### PR TITLE
ANC duplicate LMP data cleaning script

### PR DESCRIPTION
**Pull request by brian-mw**
## Context
[This request](https://app.shortcut.com/mwemr/story/330/length-of-pregnancy) prompted a change in the frontend to edit lmp  (last menstrual period) date when provider enters a different fundal height.

## Issue

The Implementation (change) made was recording new CURRENT PREGANNCY encounter on the current day and entering all new LMP info. this caused the client to have multiple LMP dates in the system and caused reporting issues

## Changes in the codebase

Developed a data cleaning tool to remove any CURRENT PREGNANCY encounters recorded after the first visit

## Changes outside the codebase

[Fix applied in the frontend](https://github.com/EGPAFMalawiHIS/HIS-Core/pull/992)

## Ticket

[Inaccurate report](https://app.shortcut.com/egpaf-2/story/94/issue-8654-cohort-report-discrepancies)

[Frontend Fix](https://app.shortcut.com/egpaf-2/story/3377/multiple-lnmp-dates-anc-issue)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,31 @@
...
-      * [Creating a Service](#creating-a-service)
-      * [Raw Data Store](#raw-data-store)
-      * [For developers](#for-developers)
-         * [Documentation](#documentation)
-            * [Building the Documentation](#building-the-documentation)
-         * [Testing](#testing)
-         * [Coding style/standards](#coding-stylestandards)
-         * [Useful (recommended) tools for development](#useful-recommended-tools-for-development)
-         * [Dev Container](#dev-container)
+- [BHT EMR API](#bht-emr-api)
```
Comment (Skipper-116) [inline README.md]: We are going with BHT EMR API?

File: bin/ANC/fix_patients_with_dup_lmp.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# rubocop:disable Metrics/MethodLength
+# frozen_string_literal: true
+
+def patients
+  ActiveRecord::Base.connection.select_all <<~SQL
+    SELECT ob.person_id, count(ob.person_id) dup_lmp_count, min(ob.obs_datetime) latest_preg_min_lmp_date
+    FROM obs ob
+            INNER JOIN(SELECT MAX(ob2.obs_datetime) start_date,
```
Comment (Skipper-116) [inline bin/ANC/fix_patients_with_dup_lmp.rb]: This query looks okay. Just format it to be easily readable.

**Review by Skipper-116 — COMMENTED** [review]
This looks good to go. Left some comments though

File: README.md (hunk 1 above)
Comment (brian-mw) [reply README.md]: lol IDK the complications of changing the name

**Review by Skipper-116 — APPROVED** [review]

**Pull request merged**